### PR TITLE
Fixed small test issues to get build working

### PR DESCRIPTION
**Pull request by pon**
@lob/api-engineering-team

### CODE REVIEW

File: tests/test_lob.py [hunk 1]
```diff
@@ -11,3 +11,6 @@ def test_bad_auth(self):
 
     def test_set_version(self):
         self.assertEqual(lob.api_version, 'apiVersion')
+
+    def tearDown(self):
```
Comment (leore) [inline tests/test_lob.py]: ahh does this delete the api version from the test above?

Comment (pon) [inline tests/test_lob.py]: Yeah - it was causing all tests before to succeed and all the ones following to fail

**Comment by leore** [comment]
LGTM

**Comment by mgartner** [comment]
LGTM

**Pull request merged**